Source/MediaInfo/TimeCode.cpp

```diff
@@ -894,6 +894,8 @@ std::string Date_MJD(uint16_t Date_)
 {
     //Calculating
     double Date = Date_;
+    // adjust the MJD to support dates beyond the 16 bit rollover in 2038
+    if ((Date_ & 0x8000) == 0) Date += 0x10000;

```

Comment:
```suggestion
    if (Date_ < 0x8000)
        Date += 0x10000; // adjust the MJD to support dates beyond the 16 bit rollover in 2038
```

Reply:
I think this is OK but EN 300 468 will say "if the high order bit of the 16bit MJD is 0, then add 0x10000".
To me `x & 0x8000 == 0` is a bit test, not a value test

Reply:
> To me x & 0x8000 == 0 is a bit test, not a value test

OK, keep it.